docs/dev/check_references.md

```diff
@@ -71,15 +71,72 @@ For more information about YAML syntax, see [Wikipedia][2]. Feel free to play ar
 
 Every integration contains a `manifest.json` file that describes operating parameters, positioning within the greater Datadog integration eco-system, and other such items.
 
-The complete list of mandatory and optional attributes for the `manifest.json` file:
+There are two versions of the `manifest.json` file. While we will continue supporting manifest version 1 for existing integrations, all **new** integrations, Datadog Apps, and Marketplace offerings should use version 2 of the manifest.
+
+If you see the following at the top of your `manifest.json` file, this means you are using version 2:
+
+```"manifest_version": "2.0.0"```
+
+Otherwise, you can assume that you are using version 1. 
+
+These two versions have different attributes and structures. You can find the complete list of mandatory and optional attributes for both versions of the `manifest.json` file below: 
+
+{{< tabs >}}
+{{% tab "Manifest Version 2" %}}
+
+| Attribute | Type  | Mandatory/Optional       | Description                                                                                                                                                                                                                                               |   |
+|------------------------------------------------------|-------------------------------------------------------------------------|-------------------------------------------|-----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|---|
+| `manifest_version`                                   | String Enum                                                             | Mandatory    | Version of the manifest schema. Supported values include `1.0.0` and `2.0.0`.                                                                                                                                                                                     |   |
+| `app_id`                                             | String                                                                  | Mandatory                                  | The unique identifying name of this offering. Usually kebab case of the display name.              |   |
```

Comment:
Small question/suggestion: 

Does it make sense to actually write in "kebab-case," or add an example for clarification? 

With the JSON object name being in snake_case, but needing the identifying name to be in kebab-case, seems like it could be helpful.

Reply:
Added an example in upcoming commit